.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+null
```

Comment:
Hey @apollo-bot (ahem, @evans) Was this `null` meant to be here?  Should I manually edit it out?